Source/Spark.swift

```diff
@@ -51,8 +51,7 @@ public class Spark {
         let deviceService = DeviceService(authenticationStrategy: authenticationStrategy)
         let callManager = CallManager(authenticationStrategy: authenticationStrategy, deviceService: deviceService)
         let webSocketService = WebSocketService(authenticationStrategy: authenticationStrategy, callManager: callManager)
-        let applicationLifecycleObserver = ApplicationLifecycleObserver(webSocketService: webSocketService, callManager: callManager, deviceService: deviceService)
```

Comment:
Yup, our fault. The behavior is reimplemented now. Ready to merge once the test pass!